Triton_Inference_Server_Python_API/examples/kafka-io/requirements.txt

```diff
@@ -0,0 +1,10 @@
+click
+confluent_kafka
+gcn-kafka
+jsonschema
+pandas
+ray
+ray[serve]
+torch
+transformers
+tritonclient
```

Comment:
I think we need to pin these dependencies, Otherwise, in the future users will be pulling an old container but using the newest packages for things like `tritonclient`. See how we do it [here](https://github.com/triton-inference-server/tutorials/blob/main/Quick_Deploy/HuggingFaceTransformers/Dockerfile#L27).

Reply:
Updated in the latest changes